frontend/src/games/view/GamePage.tsx

```diff
@@ -25,8 +35,15 @@ const GamePage = () => {
     const api = useApi();
     const request = useRequest<Game>();
     const featureRequest = useRequest();
+    const updateRequest = useRequest();
     const { cohort, id } = useParams();
     const user = useAuth().user;
+    const [searchParams, setSearchParams] = useSearchParams({
+        firstLoad: 'false',
+    });
+    const firstLoad = searchParams.get('firstLoad') === 'true';
+    const [game, setGame] = useState<Game>();
```

Comment:
Why are we adding another state variable instead of just using `request.data`?

Reply:
Thanks!